custom-per-route-options.html.md.erb

```diff
@@ -0,0 +1,92 @@
+---
+title: Configuring per-route options
+owner: CF for VMs Networking
+---
+
+By default, the communication between Gorouter and backends can only be configured at the platform level. This means that any settings affecting this communication apply universally across the entire platform.
+However, with the introduction of per-route features, developers now have the flexibility to customize this configuration on a more granular level.
+
+The ability to customize properties on a per-route basis provides greater control and adaptability, ensuring that the route can be optimized for its unique requirements.
+
+The following per-route options exist:
+- `loadbalancing`: Configures the load balancing algorithm used by Gorouter for this particular route. Possible options are: `round-robin` and `least-connections`.
+
+Available options are described in more detail in the following sections. Other options may follow in the future and will be documented here.
```

Comment:
Please remove the "future" statement.  ("Other options may follow...")
We never put future-looking statements or promises in the tech docs.